Jenkinsfile

```diff
@@ -0,0 +1,138 @@
+import groovy.json.JsonSlurper
+
```

Comment:
missing license of this file

Reply:
Thx for pointing it out. We've now changed it to the GitHub Actions, please have a review again.